.readthedocs.yaml

```diff
@@ -3,7 +3,7 @@ version: "2"
 build:
   os: ubuntu-22.04
```

Comment:
```suggestion
  os: ubuntu-24.04
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/187/commits/8906c9c7cb962fbc3324b167e866b09186f3fa8b